modules/tile-converter/src/lib/utils/statistic-utills.ts

```diff
@@ -49,16 +49,42 @@ function timeConverterFromSecondsAndMilliseconds(timeInSeconds: number, millisec
   return result;
 }
 
-export async function calculateFilesSize(params: {outputPath: string; tilesetName: string}) {
-  const {outputPath, tilesetName} = params;
+export async function calculateFilesSize(params: {
```

Comment:
Please add a one line TSDoc comment to each function stating in words what it is intended to do.

Reply:
`getSLPKTilesetTotalStorageSize()`?